<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IQBalanceOptions</class>
 <widget class="QDialog" name="IQBalanceOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>158</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSlider" name="phaseSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>180</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-500</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="gainSlider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-500</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableBalanceBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable I/Q Balance</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoBalanceBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>104</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="phaseLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phase: 0.123</string>
   </property>
  </widget>
  <widget class="QLabel" name="gainLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain: 0.123</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="IQSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>114</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I/Q Order</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
